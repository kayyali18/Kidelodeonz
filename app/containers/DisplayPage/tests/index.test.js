// import React from 'react'
// import { shallow } from 'enzyme'

import { mapDispatchToProps } from '../index'
import { RUN_STUMBLE_SAGA } from '../constants'

describe('getStumble', () => {
  let dispatch
  let mappedProps
  beforeEach(() => {
    dispatch = jest.fn()
    mappedProps = mapDispatchToProps(dispatch)
  })

  it('should map a function to getStumble', () => {
    expect(mappedProps.getStumble).toBeDefined()
  })

  it('should dispatch the stumbleSaga action', () => {
    const expected = {
      type: RUN_STUMBLE_SAGA,
    }
    mappedProps.getStumble()
    expect(dispatch).toHaveBeenCalledWith(expected)
  })
})
