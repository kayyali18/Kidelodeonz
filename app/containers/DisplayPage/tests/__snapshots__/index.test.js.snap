// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayPage /> should match the snapshot 1`] = `[Function]`;
